Core/Matcher/ContentVersionMatcher.php

```diff
@@ -107,13 +107,17 @@ public function matchContentVersions(array $versionConditions, Content $content)
         }
     }
 
-    protected function matchAnd(array $conditionsArray, $content)
+    protected function matchAnd(array $conditionsArray, $content = null)
```

Comment:
I did not even ask myself the question, why do you think it should?

Reply:
well, the abstractMatcher matchAnd don't have that typehint. 
`protected function matchAnd($conditionsArray)`

but i guess in this case we should add the typehint there instead of remove it from here. 
i'm fine with your pr.

Reply:
Ah yes, and the AbstractMatcher for the matchOr and all other functions do have the typehint, so I guess it should indeed be added one the abstract class instead of removed here.